tutorials/hist/Normalizing_Histogram.C

```diff
@@ -0,0 +1,171 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 7 different methods of normalizing Histograms
```

Comment:
We shouldn't show how many ways there are, but demonstrate just the two main, most simple interfaces:

- with `"width"` argument: `h->Scale(1./h->Integral("width");` - his should be the default
- without `"width"` argument: `h->Scale(1./h->Integral();`

Reply:
Okay, I will remove the synonymous ones

Reply:
Okay, I just kept the ones you described above, as the other ones were basically the same.